src/pages/event.js

```diff
@@ -70,59 +68,36 @@ export default function Events() {
 export function Event() {
   const params = useParams();
   const { eventId } = params;
-  const dispatch = useDispatch();
   const { trackPageView, trackLink } = useMatomo();
 
-  const tokens = useSelector((state) => state.events.tokens);
-  const loadingEvent = useSelector((state) => state.events.eventStatus);
-  const errorEvent = useSelector((state) => state.events.eventError);
-  const event = useSelector((state) => state.events.event);
-
   const [pageIndex, setPageIndex] = useState(0);
+  const [event, setEvent] = useState(undefined);
```

Comment:
I believe we can leave this as `useState()` without the `undefined`

Reply:
I would put it `null` to differentiate and just to bother a little.